casper/src/main/scala/coop/rchain/casper/reporting/ReportingCasper.scala

```diff
@@ -119,25 +122,31 @@ object ReportingCasper {
           terms: Seq[ProcessedDeploy],
           systemDeploys: Seq[ProcessedSystemDeploy],
           blockData: BlockData,
-          withCostAccounting: Boolean
+          withCostAccounting: Boolean,
+          rand: Blake2b512Random
```

Comment:
Please put _random_ parameter after systemDeploys because they are dependent for Rholang execution.